### PR TITLE
Fix blow_up for non-prime ideals

### PR DESCRIPTION
**Pull request by simonbrandhorst**

### CODE REVIEW

File: experimental/Schemes/src/IdealSheaves.jl [hunk 1]
```diff
@@ -216,6 +222,11 @@ function IdealSheaf(Y::AbsCoveredScheme,
 end
 
 
+function one_ideal_sheaf(X::AbsCoveredScheme)
+  dd = IdDict{AbsAffineScheme, Ideal}(U=>ideal(OO(U), elem_type(OO(U))[one(OO(U))]) for U in affine_charts(X))
+  return IdealSheaf(X, dd, check=false)
+end
+
```
Comment (HechtiDerLachs) [inline experimental/Schemes/src/IdealSheaves.jl]: This already exists as `unit_ideal_sheaf`. We should not duplicate it.

**Review by HechtiDerLachs — APPROVED** [review]
Looks good apart from the one duplication.

File: experimental/Schemes/src/IdealSheaves.jl (hunk 1 above)
Comment (simonbrandhorst) [reply experimental/Schemes/src/IdealSheaves.jl]: I was not aware. A search indeed supports the term `unit ideal` over `one ideal` .

**Review by afkafkafk13 — APPROVED** [review]
Looks good. Please adjust the doctests accordingly.

**Comment by simonbrandhorst** [comment]
@benlorenz  one of the booktests fails. How to handle this?

**Comment by benlorenz** [comment]
> @benlorenz one of the booktests fails. How to handle this?

I restarted the job, this was the random GC corruption error again that I am still debugging. Let's hope it succeeds this time.

**Pull request merged**